src/ui/reducers/app.ts

```diff
@@ -237,28 +264,16 @@ export const getAnalysisPointsForLocation = (
   if (!location) {
     return;
   }
-  const focusRegion = getFocusRegion(state);
+
   const key = getLocationAndConditionKey(location, condition);
   const points = state.app.analysisPoints[key];
-
-  if (focusRegion && points && points !== "error") {
-    return points.filter(p => isInTrimSpan(p.time, focusRegion));
-  }
-
-  return points;
+  return points ? getPointsInTrimSpan(state, points) : undefined;
 };
 
 export const getHoveredLineNumberLocation = (state: UIState) => state.app.hoveredLineNumberLocation;
 export const getPointsForHoveredLineNumber = (state: UIState) => {
   const location = getHoveredLineNumberLocation(state);
-  const points = getAnalysisPointsForLocation(state, location);
-  const focusRegion = getFocusRegion(state);
-
-  if (focusRegion && points && points !== "error") {
-    return points.filter(p => isInTrimSpan(p.time, focusRegion));
-  }
-
-  return points;
+  return getAnalysisPointsForLocation(state, location);
```

Comment:
no `getPointsInTrimSpan()`?

Reply:
It's applied inside `getAnalysisPointsForLocation` so no need here. Let me know if I missed something!